cmd/mempool/models/register_global_constant.go

```diff
@@ -3,14 +3,14 @@ package models
 // RegisterGlobalConstant -
 type RegisterGlobalConstant struct {
 	//nolint
-	tableName struct{} `pg:"register_global_constant"`
+	tableName struct{} `pg:"register_global_constant" comment:"register_constant operation - is used to register a global constant - Micheline expression that can be reused by multiple smart contracts."`
 	MempoolOperation
-	Source       string `json:"source"`
-	Fee          string `json:"fee"`
-	Counter      string `json:"counter"`
-	GasLimit     string `json:"gas_limit"`
-	StorageLimit string `json:"storage_limit"`
-	Value        JSONB  `json:"value"  pg:"value,type:jsonb"`
+	Source       string `json:"source"` // DISCUSS: This is sender address or constant address?
```

Comment:
This is sender address (they will soon even rename it on the proto level)
Same for all manager operations (having source field)